cardano-cli/src/Cardano/CLI/Byron/Query.hs

```diff
@@ -25,7 +25,8 @@ import           Cardano.Config.Types (SocketPath (..))
 import           Cardano.Api.LocalChainSync (getLocalTip)
 import           Cardano.CLI.Environment
                    (EnvSocketError, readEnvSocketPath, renderEnvSocketError)
-import           Cardano.TracingOrphanInstances.HardFork ()
+-- TODO @intricate: This forces us to import "cardano-node". Fix this.
+import           Cardano.Tracing.OrphanInstances.HardFork ()
```

Comment:
I need to address this.

Reply:
Lets address this after this PR is merged.

Reply:
Alright, well just remove the "TODO @intricate".

```suggestion
-- TODO: This forces us to import "cardano-node". Fix this.
import           Cardano.Tracing.OrphanInstances.HardFork ()
```

Reply:
I'll fix this in my next PR, I've been fighting with CI for some time.